pkgs/tools/security/bitwarden/cli.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, buildNpmPackage
+, nodejs-18_x
+, fetchFromGitHub
+, python3
+}:
+
+let
+  buildNpmPackage' = buildNpmPackage.override { nodejs = nodejs-18_x; };
```

Comment:
I think bitwarden-cli only builds with Node.js 18:
```json
"engines": {
  "node": "~18",
  "npm": "~9"
}
```
Thus I want to prevent it from breaking if `nodejs` starts pointing to a newer version.

Reply:
We usually just still have packages use latest nodejs until it's a problem, but also a lot of the upstreams I've dealt with only leave very outdated runtimes in there when it works fine on newer, non-EOL versions

I don't mind too much either way and nodejs_18 is still fine enough, so I'll mark this resolved